File: cpp/hello.cpp
#include <iostream>
int main(){
	int n = 5,total=0;
	int p;
	int i;
	double average;
	std::cout << "hello world \n";
	for(i=0;i<n;i++){
		std::cout << "Enter price of item : " << i + 1 << ": ";
		std::cin >> p;
		total += p;
	}
	std::cout << std::endl << "total price " << total << std::endl;
	average = (double) total / n;
	std::cout << "average price " << average << "\n";
}

File: golang/ch1/hello.go
package main

import "fmt"

func main() {
	var x = 10
	var y int
	var a float64
	z := 0.0
	y = x * 2
	z = float64(x * y)
	a = float64(x) * z
	fmt.Println("Hello, world!")
	fmt.Println("value of x", x)
	fmt.Println("value of y", y)
	fmt.Println("value of z", z)
	fmt.Println("value of a", a)
}
